crates/pop-cli/src/commands/build/mod.rs

```diff
@@ -28,9 +29,9 @@ pub(crate) struct BuildArgs {
 	/// The package to be built.
 	#[arg(short = 'p', long)]
 	pub(crate) package: Option<String>,
-	/// For production, always build in release mode to exclude debug features.
-	#[clap(short, long)]
-	pub(crate) release: bool,
```

Comment:
This is a breaking change.
Users that were running with `pop build --release` won't be able to just use this newer version. Imagine scripts.

We should consider a deprecation message and maybe an alias from `--release` to `--profile=release` during at least one more release of the CLI.

Reply:
An alias is ideal - the build command should have some familarity with cargo, allowing -r, --release or --profile release

Reply:
It should not be deprecrated in my view.

Reply:
Not a bad idea to use the alias. 
 A suggestion would be to add the [conflicts_with](https://docs.rs/clap/latest/clap/struct.Arg.html#method.conflicts_with) rule to prevent users from using both the `--release `and `--profile=production` flags at the same time.
```
#[clap(short, long, conflicts_with="profile")]
pub(crate) release: bool,
```

Reply:
I agree that it should not be deprecated for the normal build command and put it back in